File: java/abc206/A.java
import java.util.*;

public class A {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int N = sc.nextInt();
    double price = Math.floor(1.08 * (double) N);
    if (price > 206.0) {
      System.out.println(":(");
    } else if (price == 206) {
      System.out.println("so-so");
    } else {
      System.out.println("Yay!");
    }
  }

}

File: rust/abc206/src/a.rs
use proconio::input;
// use proconio::marker::Chars;
use std::io::{stdout, BufWriter, Write};

pub fn main() {
  let out = stdout();
  let mut out = BufWriter::new(out.lock());
  input! {n: u16}

  let price: u16 = ((1.08f32 * (n as f32)).floor()) as u16;
  writeln!(
    out,
    "{}",
    if price < 206u16 {
      "Yay!"
    } else if price == 206u16 {
      "so-so"
    } else {
      ":("
    }
  )
  .unwrap();
}
